guides/manual-installation.md

```diff
@@ -297,7 +317,7 @@ docker run -d \
  -e DATABASE_URL=ecto://trento_user:web_password@host.docker.internal/trento \
  -e EVENTSTORE_URL=ecto://trento_user:web_password@host.docker.internal/trento_event_store \
  -e ENABLE_ALERTING=false \
- -e PROMETHEUS_URL='http://host.docker.internal:9090' \
+ -e PROMETHEUS_URL='localhost:9090' \
```

Comment:
why this change?
this `PROMETHEUS_URL` is used by the trento web container, which as it is running in a conatiner, having `host.docker.internal` says that prometheus is running in the host machine.
With `localhost`, you are saying that prometheus is running in the same container.

Reply:
My bad, thanks! I understood it before exact in an opposite way!